packages/react-sdk/src/components/meta.ts

```diff
@@ -32,14 +32,32 @@ const meta = {
   Image,
 } as const;
 
-type MetaProps = Record<
-  string,
-  {
-    type: string;
-    required: boolean;
-    defaultValue: null | string;
-    options?: string[];
-  }
->;
+type MetaProp =
```

Comment:
In case of we switch to `ts` instead of `json` generation.
I.e. `as const`, I would be able to generate this types from ts.